src/pages/YourShoppingListsHere/Anton/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect, useRef } from 'react'
+import styled from 'styled-components'
+import { Button } from 'antd'
+import { PercentageOutlined } from '@ant-design/icons'
+import { stringify } from 'rc-field-form/es/useWatch'
```

Comment:
In addition, do order your imports according to [this guide here](https://www.notion.so/heads-of-rhdevs/WIP-RHDevs-Frontend-Code-Style-Guide-b5f294c047f14f78b370031afb2c73c5#3bba04803bae49f4ba3bdbc611c3d497)